docs/source/query.rst

```diff
@@ -41,36 +41,47 @@ Basic Expressions
 =================
 
 Filter arguments are a mapping of expressions, where a single expression consists of a key-value pair.
-All selected documents must match these expressions.
+All selected documents will match these expressions.
+
 
 The simplest expression is an *exact match*.
-For example, in order to select all jobs whose state point key *a* has the value 42, you would use the following expression: ``{'a': 42}`` as follows:
+For example, in order to select all jobs whose state point key *a* has the value 42, you would use the following expression: ``{"a": 42}`` as follows:
 
 .. code-block:: python
 
     project.find_jobs({"a": 42})
 
+.. code-block:: bash
+
+    signac find '{"a": 42}'
+
 Select All
 ----------
 
 If you want to select the complete data set, don't provide any filter argument at all.
 The default argument of ``None`` or an empty expression ``{}`` will select all jobs or documents.
-As was previously demonstrated, iterating over all jobs in a project can be accomplished directly:
+As was previously demonstrated, iterating over all jobs in a project can be accomplished directly.
 
 .. code-block:: python
 
     for job in project:
         pass
 
+On the command line, ``signac find`` without an arguments will return a list of all jobs.
+
+.. code-block:: bash
+
+    signac find
+
 .. _simple-selection:
 
 Simple Selection
 ----------------
 
-To select documents by one or more specific key-value pairs, simply provide these directly as filter arguments.
+To select documents by one or more specific key-value pairs, provide these directly as filter arguments.
 For example, assuming that we have a list of documents with values *N*, *kT*, and *p*, as such:
 
-.. code-block:: python
+.. code-block:: bash
```

Comment:
Should be Python highlighting if anything not bash. The list makes it likely that it should be none.